src/org/parosproxy/paros/extension/manualrequest/ExtensionManualRequestEditor.java

```diff
@@ -65,6 +66,7 @@
 public class ExtensionManualRequestEditor extends ExtensionAdaptor implements SessionChangedListener {
 	
 	private Map<Class<? extends Message>, ManualRequestEditorDialog> dialogues = new HashMap<>();
+	ManualHttpRequestEditorDialog httpSendEditorDialog = new ManualHttpRequestEditorDialog(true, "manual", "ui.dialogs.manreq");

```

Comment:
- `private`;
 - Declare here, but instantiate in the `initView` as before (to still not create it if there's no view).